src/themes/default/elements/icon.overrides

```diff
@@ -747,9 +842,45 @@ i.icon.venus.mars:before { content: "\f228"; }
 i.icon.mars.stroke:before { content: "\f229"; }
 i.icon.mars.alternate:before { content: "\f229"; }
 i.icon.mars.vertical:before { content: "\f22a"; }
-i.icon.mars.horizontal:before { content: "\f22b"; }
 i.icon.mars.stroke.vertical:before { content: "\f22a"; }
+i.icon.mars.horizontal:before { content: "\f22b"; }
 i.icon.mars.stroke.horizontal:before { content: "\f22b"; }
-i.icon.facebook.official { content: "\f230"; }
-i.icon.pinterest.official { content: "\f231"; }
+i.icon.asexual:before { content: "\f22d"; }
+i.icon.user.plus:before { content: "\f234"; }
+i.icon.user.times:before { content: "\f235"; }
+i.icon.user.close:before { content: "\f235"; }
+i.icon.user.cancel:before { content: "\f235"; }
+i.icon.user.delete:before { content: "\f235"; }
+i.icon.user.x:before { content: "\f235"; }
 i.icon.bed:before { content: "\f236"; }
+i.icon.yc:before { content: "\f23b"; }
+i.icon.ycombinator:before { content: "\f23b"; }
+i.icon.battery.four:before { content: "\f240"; }
+i.icon.battery.three:before { content: "\f241"; }
+i.icon.battery.two:before { content: "\f242"; }
+i.icon.battery.one:before { content: "\f243"; }
+i.icon.battery.zero:before { content: "\f244"; }
+i.icon.i.cursor:before { content: "\f246"; }
+i.icon.jcb:before { content: "\f24b"; }
+i.icon.japan.credit.bureau.card:before { content: "\f24b"; }
+i.icon.diners.club.card:before { content: "\f24c"; }
+i.icon.hourglass.outline:before { content: "\f250"; }
+i.icon.hourglass.zero:before { content: "\f250"; }
+i.icon.hourglass.one:before { content: "\f251"; }
+i.icon.hourglass.two:before { content: "\f252"; }
+i.icon.hourglass.three:before { content: "\f253"; }
+i.icon.hourglass.four:before { content: "\f254"; }
+i.icon.hand.victory:before { content: "\f25b"; }
+i.icon.tm:before { content: "\f25c"; }
+i.icon.r.circle:before { content: "\f25d"; }
+i.icon.tv:before { content: "\f26c"; }
+i.icon.500px:before { content: "\f26e"; }
+i.icon.calendar.plus:before { content: "\f271"; }
+i.icon.calendar.minus:before { content: "\f272"; }
+i.icon.calendar.times:before { content: "\f273"; }
+i.icon.calendar.check:before { content: "\f274"; }
+i.icon.factory:before { content: "\f275"; }
+i.icon.muted.user:before { content: "\f27a"; }
+i.icon.muted.user.outline:before { content: "\f27b"; }
+i.icon.edge:before { content: "\f282"; }
+i.icon.ms.edge:before { content: "\f282"; }
```

Comment:
`microsoft edge` (under Brands category) it's the "official" name, being aliases `edge` and `ms edge`.